draft-ietf-rats-uccs.md

```diff
@@ -288,15 +288,14 @@ using a Detached-Submodule-Digest or Detached EAT Bundle ({{Section 5 of -eat}})
 
 A Secure Channel which preserves the privacy of the Attester may provide
 security properties equivalent to COSE, but only inside the life-span of the
-session established.  In general, a Verifier cannot correlate UCCS received
+session established.  In general, when a privacy preserving Secure Channel is employed for conveying Evidence a Verifier cannot correlate UCCS received
```

Comment:
Why is privacy only important for Attester Verifier interactions and not for all CM exchanges?

Reply:
I'm only aware of privacy preserving attestation schemes involving attester-verifier interactions.

Reply:
> I'm only aware of privacy preserving attestation schemes involving attester-verifier interactions.

A background-check scenario, the RP may be privy to Evidence redirected by the RP to the Verifier.
Also, a RP may request (of the Verifier) to view the accepted claims rather than or in addition to Attestation Results that somehow obfuscate original claims (that are privacy sensitive).
Auditors may expect access to original claims (that are privacy sensitive) who are not the Verifier (audit trail data may be considered a CM even through it isn't described by RFC9334).

Reply:
OK, thanks.  How do you want to tweak the current text?

Reply:
...when a privacy preserving Secure Channel is employed for conveying a conceptual message the receiver cannot correlate the message with the senders of other received UCCS messages...

Reply:
WFM

Reply:
WFM as well.